README.md

```diff
@@ -105,6 +105,17 @@ larger regions?
 
 Yes because I do not worry about overlapping
 features complicating the input processing. 
+We had the capacity to exploit the assumption that features are only one nucleotide long 
+when we used the code in line 31 of bed.py where we defined the parse_line function. 
+This function is the first step in processing the bed arguments files, and it states: 
+"assert bed_line.chrom_start + 1 == bed_line.chrom_end" 
+which means that we did not have to define the start and end 
+of the features individually, the end of the feature is defined in terms of the start of the feature. 
+However, in my actual code in query_bed.py, I manually establish the start and end of the 
+feature based on the bed input which declares both, so I could remove this assertion in the function 
+definition and still use query_bed.py to process bed files with longer, more general regions. 
```

Comment:
Removing the assertions wouldn't be enough. To handle the general case, there are more ways to overlap, and we need to find all features that overlap the query region. It is not super hard to do, you can look at the picture below to see how, but it is not as trivial as the code where we really only need to check if a point is within an interval.

![overlap](https://user-images.githubusercontent.com/3206538/197321916-1d231646-6995-408e-95d1-58508f220e56.png)